<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FunctionListEditor</class>
 <widget class="QDialog" name="FunctionListEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>314</width>
    <height>249</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>5</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Function List Editor</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="example">
     <property name="text">
      <string>Color Function List [ ex: c1-99 = q1 + q2*X ]</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QListWidget" name="functionList">
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1,0,8,0,0">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Function: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="selectedName">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>=f(</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="selectedFunction">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="setButton">
       <property name="text">
        <string>Set</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="cancelButton">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="saveButton">
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QPushButton" name="selectButton">
     <property name="text">
      <string>Select</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="3">
    <widget class="QLabel" name="messageLabel">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
